test-cases/scylla-operator/kubernetes-operator-upgrade.yaml

```diff
@@ -0,0 +1,13 @@
+test_duration: 300
+
```

Comment:
add a param that defines the base version for upgrade

Reply:
Done